### ISSUE TITLE
Odd stacking behavior with mortar

### ISSUE THREAD

**Issue by CalligoMiles** [anegostudios/VintageStory-Issues#761]
**Game Version:** 1.14.5
**Platform:** Windows
**Modded:** No 

### Description
Crafting a full barrel of mortar gives you a stack of 320, but once you divide said stack you are no longer able to merge it back to more than 64.

### How to reproduce
1. Make a full barrel of mortar
2. Split the resulting stack of 320 in two or more 
3. Try to merge any of the stacks

### Expected behaviour
Consistent stacking - whether at a 320 or at a 64 cap.

### Screenshots
![2021-01-20_14-00-33](https://user-images.githubusercontent.com/19876578/105178176-fcea5580-5b27-11eb-9a39-3657b3058bba.png)

**Comment by Craluminum2413** [comment]
I have idea how to fix this.

Make it so that when you take something with a more than stack in one slot, it is stacked in your inventory if there is space, or it drops out if there is no space. 

For example, from 346 mortar in barrel you get 64 mortar when Left-click on it, and it is distributed in inventory, when you Shift-click it

**Comment by copygirl** [comment]
Yeah, either the amount of mortar you can take from a barrel in one interaction should be at most its maximum stack size, or the stack size should be increased to account for this.
